src/TraitsV2/MetaclassForTraits.class.st

```diff
@@ -76,6 +76,16 @@ MetaclassForTraits >> isTrait [
 	^ true
 ]
 
+{ #category : #'as yet unclassified' }
```

Comment:
wrong category

Reply:
I agree with the movement of these methods because they do not create any real dependency